src/config/AT-START-F435/config.h

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Directory name should match the target name.

Reply:
Done